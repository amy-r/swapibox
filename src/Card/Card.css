@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700');

.Card {
  background-color: #D6D5D1;
  /*border: 3px solid #D53B24;*/
  border-radius: 10px;
  color: #374157;
}

.body { 
  background-color: black; 
}

h3 {
  font-family: 'Lato', sans-serif;
  letter-spacing: 3px;
  text-transform: uppercase;
  margin: auto;
  text-align: center;
  font-weight: 800;
}

ul{
  list-style: none;
}