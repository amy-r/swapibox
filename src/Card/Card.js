import React, { Component } from 'react';


class Card extends Component {
  constructor(props){
    super(props);
    this.state = {

    }
  }

  render() {
    return (
      <h3> Name </h3>
    )
  }
}

export default Card;