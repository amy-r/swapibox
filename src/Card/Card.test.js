 /* eslint-disable */ 
import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

describe( 'Card', () => {

  it('exists', () => {
  })

  it('renders without crashing', () => {
    const renderedComponent = shallow(<Card />, {disableLifecycleMethods: true})
    expect(renderedComponent).toMatchSnapshot();
  })

});