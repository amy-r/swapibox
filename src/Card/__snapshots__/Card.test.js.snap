// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ul>
        
</ul>,
      "className": "Card",
      "id": undefined,
      "onClick": undefined,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [],
      },
      "ref": null,
      "rendered": Array [],
      "type": "ul",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ul>
          
</ul>,
        "className": "Card",
        "id": undefined,
        "onClick": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [],
        },
        "ref": null,
        "rendered": Array [],
        "type": "ul",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
